app/components/user-cards.js

```diff
@@ -1,40 +1,75 @@
 import Component from '@ember/component';
-import { computed } from '@ember/object';
+import EmberObject, { computed } from '@ember/object';
+
 
 export default Component.extend({
 
     selected_option:null,
+    sort_option:null,
+    sort_order:null,
+    sortfieldList:null,
+    sortorderlist:null,
     
-    teamslist : computed ('selected_option',function(){
-        if(this.selected_option!='All Employees')
-            return this.get('members').filterBy('team',this.get('selected_option'))
-        else
-            return this.get('members').filterBy('team')
-        
+    teamslist : computed ('selected_option','sort_option','sort_order',function(){
+
+        let teamlistArray = this.get('members').filterBy('team',this.get('selected_option')).sortBy(this.get('sort_option'));
+        let allteamlistArray = this.get('members').filterBy('team').sortBy(this.get('sort_option'))
```

Comment:
Here too, we do sorting first and again doing reverse sorting in the else part. Can you check and handle?